afu_isboden_pub/build.gradle

```diff
@@ -8,6 +8,9 @@ task transferAfuIsboden(type: Db2DbTask){
     transferSets = [
             new TransferSet('afu_isboden_pub_bodeneinheit_wald.sql', 'afu_isboden_pub.bodeneinheit_wald', true),
             new TransferSet('afu_isboden_pub_bodeneinheit_lw.sql', 'afu_isboden_pub.bodeneinheit_lw', true),
-            new TransferSet('afu_isboden_pub_erosionsgefahr.sql', 'afu_isboden_pub.erosionsgefahr', true)
+            //new TransferSet('afu_isboden_pub_erosionsgefahr.sql', 'afu_isboden_pub.erosionsgefahr', true),
```

Comment:
Wieso haben wir die erosionsgefahr noch mal auskommentiert?

Reply:
erosionsgefahr wurde auskommentiert, da inhaltlich identisch mit bodenerosion. Originaldaten liegen in public.afu_erosion_gbf20_p.